ClientApp/src/Pages/Home/Home.tsx

```diff
@@ -3,35 +3,35 @@ import './Home.scss';
 
 function Home() {
   return (
-    <div>
-      <h1>Welcome to the World of Whale Watching!</h1>
+    <div className="content-container">
+      <h1 className="title">Welcome to the World of Whale Watching!</h1>
```

Comment:
It looks like we're using very generic classes to be used on all other pages, might be a good idea to give a heads up to everyone to use these same classnames when they do styling on their pages :)